main_test.go

```diff
@@ -92,9 +92,9 @@ func (t *MainTest) TestStringifyShouldReturnAllFlagsPassedInMountConfigAsMarshal
 		},
 	}
 
-	actual := util.Stringify(mountConfig)
+	actual, _ := util.Stringify(mountConfig)
```

Comment:
We should not ignore the error. Instead we should assert as nil.

Reply:
This was the original behaviour before my changes, so I decided to keep it as it is without playing around with them.
Anyway, I'll assert on every error now.